scripts/commit_classifier.py

```diff
@@ -597,10 +606,8 @@ def classify(
         self.update_commit_db()
 
         if phabricator_deployment is not None and diff_id is not None:
```

Comment:
Now we could check if `self.revision` has a value instead of this. Also, we could drop `phabricator_deployment` and `diff_id` from the arguments of the `classify()` method since they are not needed anymore (the assertions could be moved to the constructor).

Reply:
Well noted